src/app/modules/group/http-services/remove-group.service.ts

```diff
@@ -0,0 +1,36 @@
+import { Injectable } from '@angular/core';
+import { forkJoin, Observable, of } from 'rxjs';
+import { HttpClient } from '@angular/common/http';
+import { appConfig } from '../../../shared/helpers/config';
+import { SimpleActionResponse } from '../../../shared/http-services/action-response';
+import { catchError, map } from 'rxjs/operators';
+
+export function parseResults(data: Map<string, SimpleActionResponse>): { countRequests: number, countSuccess: number, errorText?: string } {
```

Comment:
`{ countRequests: number, countSuccess: number, errorText?: string }` looks exactly like the type `Result` from group-removal-response-handling.ts, shouldn't it be imported from there ?

Reply:
Fixed